inference-engine/src/vpu/common/src/ngraph/transformations/dynamic_to_static_shape.cpp

```diff
@@ -69,6 +69,23 @@ bool propagateUpperBoundFromExistingDSR(std::shared_ptr<ngraph::Function>& funct
     return function_changed;
 }
 
+void checkDynamicFunction(const ngraph::Function& function) {
+    for (auto const& split : function.get_ordered_ops()) {
```

Comment:
What do you think about extracting it into separate function like `validateSplit`?

Reply:
Don't see a reason while there is no any other operation specific check.

Reply:
Ok, got it.